d4s-test/src/main/scala/d4s/test/envs/DynamoTestEnv.scala

```diff
@@ -3,15 +3,27 @@ package d4s.test.envs
 import d4s.DynamoDDLService
 import d4s.config.DynamoConfig
 import d4s.test.envs.DynamoTestEnv.DDLDown
-import distage.{DIKey, ModuleDef}
+import distage.{Activation, DIKey, ModuleDef}
 import izumi.distage.docker.Docker
 import izumi.distage.docker.modules.DockerContainerModule
+import izumi.distage.model.definition.StandardAxis.Env
 import izumi.distage.model.definition.{DIResource, Module}
+import izumi.distage.testkit.TestConfig
+import izumi.distage.testkit.scalatest.DistageBIOSpecScalatest
 import logstage.LogBIO
 import net.playq.aws.tagging.AwsNameSpace
 import zio.{IO, Task}
 
-trait DynamoTestEnv {
+trait DynamoTestEnv extends DistageBIOSpecScalatest[IO] {
```

Comment:
```scala
trait DynamoTestEnv[F[_]] { this: DistageAbstractScalatestSpec[F] }
```
We don't want to force `zio.IO`, distage-testkit works with any effect monad

Reply:
Done ✅